### PR TITLE
add noetic test and also checks python 2/3 compatibility

### PR DESCRIPTION
**Pull request by k-okada**
this PR changes simply update `print x` to `print (x)`. Should we use `rospy.logerr` or `rospy.logout` as we do in #20 ?

### CODE REVIEW

File: src/app_manager/app_manager.py [hunk 1]
```diff
@@ -33,8 +33,13 @@
 # Revision $Id: app_manager.py 14948 2011-09-07 19:25:54Z pratkanis $
 
 # author: leibs
+import sys
+
+if sys.version_info[0] == 3:
+    import _thread # python3 renamed from thread to _thread
```
Comment (knorth55) [inline src/app_manager/app_manager.py]: just importing does not seem to solve the problem.
maybe `import _thread as thread`?

**Review by knorth55 — CHANGES_REQUESTED** [review]

**Comment by knorth55** [comment]
I think it is better to use `rospy.logerr` and `rospy.loginfo` for logging instead of `print` function.

File: src/app_manager/exchange.py [hunk 2]
```diff
@@ -40,6 +40,8 @@
 avialable and removable applications.
 """
 
+from __future__ import print_function
```
Comment (v4hn) [inline src/app_manager/exchange.py]: Either you add it to *every* file or you leave it out.

File: src/app_manager/master_sync.py [hunk 3]
```diff
@@ -1,6 +1,13 @@
+import sys
+
+if sys.version_info[0] == 3:
+    from urllib.parse import urlparse # python3 renamed from thread to _thread
```
Comment (v4hn) [inline src/app_manager/master_sync.py]: The comment is wrong

**Review by v4hn — COMMENTED** [review]
I would strictly prefer migration to `rospy.log*`.
These scripts were written in early stages of ROS development, and apparently the logging commands were not as established back then.

**Comment by v4hn** [comment]
ping @k-okada.

This is still required for noetic.

**Comment by k-okada** [comment]
Sorry for late, There seems  several place to update (use `rospy.log`, fix comment and so on..) Could @v4hn or @knorth55 create PR for me?

**Comment by v4hn** [comment]
@knorth55 could you create a follow-up pr to fix this up?

**Comment by knorth55** [comment]
@k-okada @v4hn I made a follow-up PR https://github.com/k-okada/app_manager/pull/1

**Review by v4hn — APPROVED** [review]

**Pull request merged**